### DISCUSSION TITLE
8-0 Part 2, anyone else have an error when downloading?

### DISCUSSION THREAD

**Discussion by jenw-ua** [ua-gist604b-f24/syllabus#43] [General]
I'm getting an error at Step 2, when downloading and entering :
wget https://www.naturalearthdata.com/http//www.naturalearthdata.com/download/50m/raster/NE1_50M_SR_W.zip
unzip NE1_50M_SR_W.zip

Error 500: Internal Server Error.

Anyone else run in to this or know a work around? Can anyone confirm that this line and site URL in the command is actually correct here and there isn't a typo? Can I just manually download this this and put it in the proper place (wherever that is going to be)?

wget https://www.naturalearthdata.com/http//www.naturalearthdata.com/download/50m/raster/NE1_50M_SR_W.zip

I think I see the actual data the assignment is talking about is here: https://www.naturalearthdata.com/downloads/50m-raster-data/50m-natural-earth-1/ 

<img width="1320" alt="Screenshot 2024-11-29 at 9 40 13 PM" src="https://github.com/user-attachments/assets/81e1c76d-e2fd-4c87-a8da-3ad8c01516ca">

Thanks (forgive all the hyperlinks...I can't figure out how to turn them off)

**Comment by robertgamboa10** [comment]
I was getting the same error, so I just downloaded the zip file from the Natural Earth data website. It's the link that you have above and after it downloads, I just dragged it into the Explorer area on the left. After doing that, then you can run the unzip NE1_50M_SR_W.zip command in the terminal. After it unzips, it will create a new directory in the explorer area on the left, NE1_50M_SR_W. I included a couple screenshots, hope that helps. 

<img width="1083" alt="Screenshot 2024-11-30 at 6 57 26 AM" src="https://github.com/user-attachments/assets/65fd2b10-490b-42d7-9930-2e080d43466e">





<img width="1081" alt="Screenshot 2024-11-30 at 7 01 46 AM" src="https://github.com/user-attachments/assets/f0112d5b-0d4a-41fc-8b59-9c4c862846d4">

**Reply by jenw-ua** [reply]
Thanks so much! That worked :)

**Reply by aaryno** [reply]
This is sage advice. Links change and the way this tutorial was written it appears the link now has a double hostname in it. Notice the double www.naturalearthdata.com url in the instructions:
https://www.naturalearthdata.com/http//www.naturalearthdata.com/download/50m/raster/NE1_50M_SR_W.zip
